.github/workflows/ci-title-lint-check.yaml

```
@@ -0,0 +1,82 @@
+name: CI Check Title
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+jobs:
+  title-lint:
+    name: Validate PR title
+    if: github.event.changes.title
+    runs-on: ubuntu-latest
+    steps:
+      - name: CI Check Title
+        uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          wip: true
+          # Configure which types are allowed (newline-delimited).
+          # Default: https://github.com/commitizen/conventional-commit-types
+          types: |
+            fix
+            feat
+            chore
+            build
+            release
+          # Configure which scopes are allowed (newline-delimited).
+          # These are regex patterns auto-wrapped in `^ $`.
+          scopes: |
+            core
+            ci
+            build
+            infra
+            ui
+            communicator
+            oauth
+            message-provider
+            url-state-provider
+            url-state-router
+            utils
+            config
+            juno
+            k8s
+            example
```

We're missing `template` for changes to the template app.
```suggestion
            example
            template
```